### ISSUE TITLE
500 Error but not error stack in console log

### ISSUE THREAD

**Issue by CrossLee** [actframework/actframework#1358]
controller:

```java
@Inject
PlatformOrderService

@GetAction("/test")
public void test() {
}
```

service:
```java
public interface PlatformOrderService {
    public PlatformOrderVO creatOrder(String stationId);

    public String callbackUpdateOrder(String out_trade_no);

    public PrinterAndBellContentPojo printBellContent(String out_trade_no);
}
```

There is no problem starting the app. However when it requests `/test` the server returns a 500 error page which is okay. The problem is there is no console log for the error, it makes it super hard in diagnosing the issue.

**Comment by greenlaw110** [comment]
The issue is brought in by the change for https://github.com/actframework/actframework/issues/1341

**Comment by greenlaw110** [comment]
The issue is fixed by #dfe230ec
